### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by KennethEnevoldsen**
As discussed on slack we moved the team to the website. Please let me know if I am missing any collaborators (adding everyone as reviewer to make sure that people know that their name (and email) is put here).

### CODE REVIEW

**Review by peter-sk — APPROVED** [review]
Looks great :-)

**Review by saattrupdan — APPROVED** [review]
Looks good!

File: docs/index.md [hunk 1]
```diff
@@ -37,7 +37,28 @@ Furthermore, during data processing and training, the data is stored on UCloud w
...
+  - Per Møldrup Dalum (per@cas.au.dk)
+  - Kristoffer Laigaard Nielbo (kln@cas.au.dk)
+
+
+- From [the Alexandra Institute](https://alexandra.dk/):
+  - Peter Bjørn Jørgensen (peter.jorgensen@alexandra.dk)
+  - Rasmus Larsen (rasmus.larsen@alexandra.dk)
+  - Dan Saattrup Nielsen (dan.nielsen@alexandra.dk)
+
+- From [the Center for AI Science and Applicaitons at the University of Southern Denmark](https://sdu.dk/casa):
```
Comment (HLasse) [inline docs/index.md:59]:
```suggestion
- From [the Center for AI Science and Applications at the University of Southern Denmark](https://sdu.dk/casa):
```

File: docs/index.md [hunk 2]
```diff
@@ -37,7 +37,28 @@ Furthermore, during data processing and training, the data is stored on UCloud w
...
+The Danish Foundations models collaborate with the [Danish Data Science Community](https://danskdatascience.dk/), [Centre for Humanities Computing Aarhus](https://chcaa.io/), [The Alexandra Institute](https://alexandra.dk), and [Center for AI Science and Applications](https://sdu.dk/casa) to promote the development of Danish language tools. We continually gather information about how to improve the Danish language technologies and how to best support the community. To this end we have created a [list of missing pieces for Danish NLP](https://github.com/centre-for-humanities-computing/danish-foundation-models/discussions/categories/missing-pieces-for-danish-nlp) and we invite any 1) to add to the list, 2) solve one of the problems or 3) upvote the problems you think are most important.
+
+
+## The Team
+
+- From [the Center for Humanities Computing at Aarhus University](https://chc.au.dk/):
+  - Kenneth Enevoldsen (kenneth.enevoldsen@cas.au.dk)
+  - Lasse Hansen (lasse.hansen@clin.au.dk)
+  - Martin Bernstorff (martinbernstorff@gmail.com)
```
Comment (HLasse) [inline docs/index.md:48]: Want to use your clin email as well @MartinBernstorff or prefer gmail?

**Review by HLasse — APPROVED** [review]
Looks good! Should we add Martin Carsten Nielsen and Søren Vejlgaard Holm from Alvenir as well? They've contributed a lot to the speech models and Martin seems to be consistently present at the meetings as well

File: docs/index.md (hunk 2 above)
Comment (KennethEnevoldsen) [reply docs/index.md:48]: I used the mail on notion, but @MartinBernstorff feel free to change it

**Comment by KennethEnevoldsen** [comment]
merging in for now. We can always change it

**Review by sorenmulli — APPROVED** [review]

**Pull request merged**